build/MSBuildExtensions.targets

```diff
@@ -116,6 +116,11 @@
 
       <!-- Use only major and minor in target framework version -->
       <_NETCoreAppTargetFrameworkVersion>$(_NETCoreAppPackageVersion.Split('.')[0]).$(_NETCoreAppPackageVersion.Split('.')[1])</_NETCoreAppTargetFrameworkVersion>
+      <!--
+        Currently 2.2 Microsoft.NETCore.App package is still using netcoreapp2.1 as the TFM so we need to bump down the TFM
+        This is temporary until we move the netcoreapp2.2 as the TFM.
+      -->
```

Comment:
Is there a bug tracking the TFM change?

Reply:
https://github.com/dotnet/core-setup/issues/4055. I also plan to file an issue in cli to remove this hack once I verify that it fully fixes this update of the runtime.